src/pages/NavBar.tsx

```diff
@@ -263,7 +264,6 @@ function NavBar() {
           </Toolbar>
         </Container>
       </AppBar>
-      <Toolbar />
```

Comment:
I reason I added this extra Toolbar was content of other pages was going behind the navbar, so its like extra fragment that makes other content to start by leaving that extra height at top, but I see instead you have added padding to all other content (EXCEPT HOMEPAGE) and I also see the issue if we have this toolbar with lesson page animation so I think this should be fine

Reply:
So add padding to Home page as well